index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <section id="suzhouhe">
+        <h1>
+          Suzhou River
+        </h1>
```

Comment:
For the purposes of this project, your card should start with an `h2` element -- since we're imagining the card is one of many on a page, there would be some other `h1` on the page that is the title for all the cards ("Romantic movies," for example); you don't have to add this `h1` into your code, but you should make choices for your card component knowing that it would exist.

Let me know if this makes sense or if I can help explain this further.